rpm-mgmt/artifactory.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+export ARTIFACTORY_USERNAME=dev-deployer
+export ARTIFACTORY_PASSWORD=dev@guavus
+
+export version=`cat ../VERSION`
+
+MAJOR_VERSION=$(echo $version | awk -F. '{print $1}')
+MINOR_VERSION=$(echo $version | awk -F. '{print $2}')
+PATCH_VERSION=$(echo $version | awk -F. '{print $3}')
+
+export RPM_ARTIFACTORY=artifacts.ggn.in.guavus.com:4245/ggn-dev-rpms/guavus/superset/release/
```

Comment:
can we generalize this in a way that it could read the artifactory path from somewhere else?